nodes/src/nodes/Sql.ExecuteBulkInsert.ts

```diff
@@ -0,0 +1,62 @@
+import { SqlQueryResult } from '@nodescript/adapter-sql-protocol';
+import { ModuleCompute, ModuleDefinition } from '@nodescript/core/types';
+
+import { BulkInsertBuilder } from '../lib/BulkInsertBuilder.js';
+import { requireConnection, SqlConnection } from '../lib/SqlConnection.js';
+
+export type BulkInsertParams = {
+    connection: SqlConnection;
+    tableName: string;
+    rowData: Record<string, any>[];
+};
+
+type R = Promise<SqlQueryResult>;
+
+export const module: ModuleDefinition<BulkInsertParams, R> = {
+    version: '0.0.1',
+    moduleName: 'SQL / Execute Bulk Insert',
+    description: 'Takes an array of objects representing row data and executes a SQL INSERT parameterized query.',
+    keywords: ['sql', 'database', 'insert', 'bulk', 'query'],
```

Comment:
Btw these are used when searching, so shouldn't be overused. Basically, remove the ones that already exist in module name, and only add if there are relevant synonyms. Otherwise those will popup in unexpected searches